debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorTask.java

```diff
@@ -105,7 +113,10 @@ public ChangeEventSourceCoordinator<MongoDbPartition, MongoDbOffsetContext> star
                     taskContext.filters().collectionFilter()::test,
                     DataChangeEvent::new,
                     metadataProvider,
-                    schemaNameAdjuster);
+                    schemaNameAdjuster,
+                    signalProcessor);
+
+            dispatcher.getSignalingActions().forEach(signalProcessor::registerSignalAction);
```

Comment:
WDYT to move this in the BaseSourceTask, loading it in the constructor and exposing with getSignalingActions ? Then I'll work on loading with service loader with the follow-up task.

Reply:
@mfvitale ok

Reply:
Seems that this cannot be moved into BaseSourceTask and maybe also the loading with service loader since the sinaling actions requires to have the dispatcher instance passed. So I think it requires more thinking.

Reply:
@mfvitale ok, let's make it a follow-up task